packages/icon/src/react/__specs__/index.spec.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
```

Comment:
i added these tests...not sure they're that valuable. i wanted to make sure each icon still rendered without errors....